packages/shared/styles/components/SfSteps.scss

```diff
@@ -0,0 +1,104 @@
+@import "../variables";
+
+// $component__block-property: value;
+
+$sf-steps__active-line-height-mobile: 0.1875rem;
+$sf-steps__active-line-height-desktop: 0.125rem;
+$sf-steps__active-line-color: #1d1f22;
+$sf-steps__default-border: 0.0625rem solid #f1f2f4;
```

Comment:
@DonNicoJs @filrak maybe we should use px for sizes between 1 - 10?

Reply:
@filrak what is your input here? I would go with rem for consistency but is up to you